README_WINDOWS.rst

```diff
@@ -6,18 +6,20 @@ Build Requirements
 
 - VC 14.0 Compiler
 - OpenSSL Library (https://slproweb.com/products/Win32OpenSSL.html)
-- Additional Python Packages:
-
-  - psutils
-
```

Comment:
Yay, no more psutils dependency!

Reply:
Good. :)

But: this is a unclean changeset. Changeset topic is "directory handling", but you also updated the docs with unrelated changes.

Reply:
Sorry about that. This should have been commited with the previous commit which replaced psutils with the cython implementation.